bin/bulk_load_terms_file.py

```diff
@@ -0,0 +1,37 @@
+import json
+import os
+import yaml
+
+from dictionaryutils import load_yaml
+from utils import add_codes
+
+
+
+path = "../../gdcdictionary/schemas/_terms.yaml"
```

Comment:
Shouldn't this path be ../../datadictionary/gdcdictionary/schemas/_terms.yaml?

Reply:
"../../gdcdictionary/schemas/_terms.yaml" is the correct path on my machine, since the original guidance from CTDS was to clone dictionary utils within the datadictionary folder. However that is not a requirements anymore, so I'll make this a .env configuration.